action/receipt.go

```diff
@@ -119,6 +119,18 @@ func (receipt *Receipt) Hash() hash.Hash256 {
 	return hash.Hash256b(data)
 }
 
+// AddLogs add log to receipt and filter out nil log.
+func (receipt *Receipt) AddLogs(logs ...*Log) {
```

Comment:
directly use []*Log? so no need to do logs... outside

Reply:
i like this. this is more flexible to add single log